packages/TelegramClient-UI.package/TCUEmojiHelper.class/class/findLongestRun..st

```diff
@@ -0,0 +1,10 @@
+utility
+findLongestRun: aCollection
+
+	(aCollection size > 10) ifTrue: [
+		^ self findLongestRun: (aCollection first: 10)
+	].
+	(aCollection size < 2 or: [self emojiExists: (self joinRun: aCollection)]) ifTrue: [
```

Comment:
Why 10?

Reply:
The 2 is there so we don't have to write `aCollection size = 0 or: [aCollection size = 1] or: ...`. Its just to stop the recursion for a length of 0 or 1.